gun.cpp

```diff
@@ -0,0 +1,36 @@
+// Object.cpp
+// Sheng Fang, Rabiya Majeed, & Zulaikha Zakiullah
+// This instantiates the gun class, the gun [|87.
+
+#include "gun.h"
+
+Gun::Gun() {
+	Gun::ammo = 0;
+	Gun::durability = 0;
+	Gun::length = 0;
+	Gun::width = 0;
+	Gun::shooting_speed = 0;
+	Gun::accuracy = 0;
+	Gun::durability = 0;
+	Gun::recoil = 0;
+}
+
+Gun::~Gun() {}
+
+int Gun::get_ammo() { return Gun::ammo; }
+int Gun::get_durability() { return Gun::durability; }
+int Gun::get_length() { return Gun::length; }
+int Gun::get_width() { return Gun::width; }

```

Comment:
What's the difference between length and width? If it's referring to the dimensions of the gun, the object class already has width and height property so the gun class will inherit from it, so this isn't needed. (unless this is referring to something else)

Reply:
okai! this was just gun dimensions so i will take out